src/main/java/jenkins/plugins/publish_over_ssh/descriptor/BapSshHostConfigurationDescriptor.java

```diff
@@ -81,12 +81,16 @@ public FormValidation doCheckTimeout(@QueryParameter final String value) {
         return FormValidation.validateNonNegativeInteger(value);
     }
 
+    @RequirePOST
```

Comment:
No side effects, so not strictly necessary, but doesn't hurt either assuming the UI controls send POST.